packages/schema-blocks/tests/blocks/RequiredBlocks.test.ts

```diff
@@ -1,11 +1,11 @@
 import { BlockInstance, createBlock } from "@wordpress/blocks";
 
-import renderer from "react-test-renderer";
-import { shallow } from "enzyme";
+import * as renderer from "react-test-renderer";
+import { mount } from "enzyme";
 
 import { RequiredBlock } from "../../src/instructions/blocks/dto";
 import RequiredBlocks from "../../src/blocks/RequiredBlocks";
-import { insertBlockToInnerBlock } from "../../src/functions/innerBlocksHelper";
+import { insertBlocks } from "../../src/functions/innerBlocksHelper";
```

Comment:
```suggestion
import { insertBlock } from "../../src/functions/innerBlocksHelper";
```
Plus, there are some other references in this file that still reference `insertBlocks`.
Also, the mock of `insertBlockToInnerBlock` should be changed to mock `insertBlock` instead.